target/linux/mpc85xx/files/arch/powerpc/boot/dts/t30-w.dts

```diff
@@ -0,0 +1,396 @@
+			tbi1: tbi-phy@16 {
+				reg = <0x10>;
+			};
+		};
+		
+		mdio@26000 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			compatible = "fsl,etsec2-tbi";
+			reg = <0x26000 0x1000 0xb1030 0x4>;
+		};
+
+		enet0: ethernet@b0000 {
+			compatible = "fsl,etsec2";
+			phy-connection-type = "rgmii-id";
+			fixed-link {
+				speed = <1000>;
+				full-duplex;
+			};
+		};
+
+		enet1: ethernet@b1000 {
+			status = "disabled";
+		};
+
+		enet2: ethernet@b2000 {
+			status = "disabled";
+		};
+
+		gpio0: gpio@f000 {
+			compatible = "fsl,mpc8572-gpio", "fsl,pq3-gpio";
+			reg = <0xf000 0x1000>;
+			interrupts = <47 2>;
+			interrupt-parent = <&mpic>;
+			#gpio-cells = <2>;
+			gpio-controller;
+		};
+	};
+	
+	leds {
+		compatible = "gpio-leds";
+
+		attn_orange: attn_orange {
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+			label = "orange:attn";
+		};
+		
+		status_red: status_red {
+			gpios = <&gpio0 3 GPIO_ACTIVE_LOW>;
+			label = "red:status";
+		};
+		
+		mode_green: mode_green {
+			gpios = <&gpio0 4 GPIO_ACTIVE_LOW>;
+			label = "green:mode";
+		};
+		
+		wap_green: wap_green {
+			gpios = <&gpio0 5 GPIO_ACTIVE_HIGH>;
+			label = "green:wap";
+			default-state = "keep";
+		};
+		
+		wap_orange: wap_orange {
+			gpios = <&gpio0 6 GPIO_ACTIVE_HIGH>;
+			label = "orange:wap";
+			default-state = "keep";
+		};
+		
+		failover_green: failover_green {
+			gpios = <&gpio0 7 GPIO_ACTIVE_LOW>;
+			label = "green:failover";
+		};
+	};
+
+	pci0: pcie@ffe09000 {
+		cell-index = <0x01>;
+		compatible = "fsl,mpc8548-pcie";
+		device_type = "pci";
+		#interrupt-cells = <0x01>;
+		#size-cells = <0x02>;
+		#address-cells = <0x03>;
+		reg = <0x00 0xffe09000 0x00 0x1000>;
+		bus-range = <0x00 0xff>;
+		ranges = <0x2000000 0x0 0xa0000000 0x0 0xa0000000
+					0x0 0x20000000>,
+				 <0x1000000 0x0 0x0        0x0 0xffc10000
+					0x0 0x10000>;
+		clock-frequency = <0x1fca055>;
+		interrupt-parent = <&mpic>;
+		interrupts = <0x10 0x02>;
+		interrupt-map-mask = <0xf800 0x00 0x00 0x07>;
+		interrupt-map = <0x00 0x00 0x00 0x01 0x02 0x04 0x01 0x00 0x00 
+			0x00 0x02 0x02 0x05 0x01 0x00 0x00 0x00 0x03 0x02 0x06 0x01 
+			0x00 0x00 0x00 0x04 0x02 0x07 0x01>;
+
+		pcie@0 {
+			reg = <0x00 0x00 0x00 0x00 0x00>;
+			#size-cells = <0x02>;
+			#address-cells = <0x03>;
+			device_type = "pci";
+			ranges = <0x2000000 0x0 0xa0000000 0x2000000 0x0 0xa0000000
+						0x0 0x20000000>,
+					 <0x1000000 0x0 0x0        0x1000000 0x0 0x0
+						0x0 0x100000>;
+		};
+	};
+
+	pci1: pcie@ffe0a000 {
+		cell-index = <0x02>;
+		compatible = "fsl,mpc8548-pcie";
+		device_type = "pci";
+		#interrupt-cells = <0x01>;
+		#size-cells = <0x02>;
+		#address-cells = <0x03>;
+		reg = <0x00 0xffe0a000 0x00 0x1000>;
+		bus-range = <0x00 0xff>;
+		ranges = <0x2000000 0x0 0x80000000 0x0 0x80000000 
+					0x0 0x20000000>,
+				 <0x1000000 0x0 0x0        0x0 0xffc00000 
+					0x0 0x10000>;
+		clock-frequency = <0x1fca055>;
+		interrupt-parent = <&mpic>;
+		interrupts = <0x10 0x02>;
```

Comment:
That 0x02 is likely a Interrupt flag. Could it be IRQ_TYPE_EDGE_FALLING?

Reply:
What diagnosis should I do on this ?